### PR TITLE
Update prefix table, add title to user guide

### PR DESCRIPTION
**Pull request by jweeyh**

### CODE REVIEW

**Review by 4ndrelim — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,12 +3,31 @@ layout: page
...
+    3. [Leave commands](#43-leave-commands)
+    4. [General commands](#44-general-commands)
+    5. [Data storage](#45-data-storage)
+    6. [Command summary](#46-command-summary)
+5. [FAQs](#5-faq)
 ---
 
-# **1. Introduction**
+
+# 1. Introduction
```
Comment (kennethk-1201) [inline docs/UserGuide.md:30]: make it bold for consistency

File: docs/UserGuide.md [hunk 2]
```diff
@@ -3,12 +3,31 @@ layout: page
 title: User Guide
 ---
 
-* Table of Contents
```
Comment (kennethk-1201) [inline docs/UserGuide.md:6]: put back table of contents

**Review by kennethk-1201 — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md [hunk 3]
```diff
@@ -303,7 +322,7 @@ Examples:
 
 
 
-### 4.1.5. Find employee by ID: `feid`
+### 4.1.5. Find employees by ID: `feid`
```
Comment (kennethk-1201) [inline docs/UserGuide.md:325]: should be "employee" because only one employee has the ID

File: docs/UserGuide.md [hunk 4]
```diff
@@ -179,18 +179,20 @@ There is currently no prefix for **KEYWORD** and **OLD_DEPARTMENT_NAME**.
...
 | **EMPLOYEE_ID**         | eid/                 | Similar to ID, The EMPLOYEE_ID represents the ID of an employee. <br/> It is used for any commands that require referencing an employee. <br/> It follows the same constraints as ID.                                                                                                                                                                                                                  {:/}                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```
Comment (kennethk-1201) [inline docs/UserGuide.md:187]: need to remove

**Review by kennethk-1201 — CHANGES_REQUESTED** [review]

**Review by kennethk-1201 — APPROVED** [review]

**Pull request merged**